kanopi/rooseveltedu
# 26122791: Template - Header

## Description
As a user of the Roosevelt site, I need an easy and intuitive way to navigate, search and have the same look and feel through the website, so I can quickly find what I’m looking for.

## Acceptance Criteria
- [ ] Logo should be on the header (Desktop & Mobile)
- [ ] The main menu can have up to two levels w/group title for each submenu (Desktop & Mobile)
- [ ] The main menu should act as mega-menu (Desktop)
- [ ] Utility menu has only one level (Desktop & Mobile)
- [ ] Search option should expand/collapse when clicked - refer to design (Desktop/Mobile)
- [ ] Sticky header (Desktop and Mobile)
- [ ] Mobile merge main menu and utility menu (Mobile)
- [ ] Mobile stacks menu option, logo and search (Mobile)
- [ ] Header has to match the design (Desktop & Mobile)
- [ ] Accessibility check

## Affected URL
[Full site](http://pr-71-rooseveltedu.pantheonsite.io/)

## Related Tickets
* [Template - Header - Desktop - Logo](https://kanopi.teamwork.com/#/tasks/26122795)
* [Template - Header - Mobile - Logo](https://kanopi.teamwork.com/#/tasks/26124946)
* [Template - Header - Desktop - Main Menu](https://kanopi.teamwork.com/#/tasks/26122796)
* [Template - Header - Mobile - Main Menu](https://kanopi.teamwork.com/#/tasks/26124947)
* [Template - Header - Mobile - Utility Menu](https://kanopi.teamwork.com/#/tasks/26124948)
* [Template - Header - Desktop - Utility Menu](https://kanopi.teamwork.com/#/tasks/26122797)
* [Template - Header - Desktop - Search Widget](https://kanopi.teamwork.com/#/tasks/26122799)
* [Template - Header - Mobile - Search Widget](https://kanopi.teamwork.com/#/tasks/26124950)
* [Template - Header - templating/style miscellaneous](https://kanopi.teamwork.com/#/tasks/26124945)
* [Template - Header - Desktop - Sticky menu & transition](https://kanopi.teamwork.com/#/tasks/26122800)
* [Template - Header - Mobile - Sticky menu & transition](https://kanopi.teamwork.com/#/tasks/26124951)

## Steps to Validate
1. Navigate to website
1. See global header
1. Click on the search icon to expose search bar
1. Click on the hamburger icon to expose megamenu

## Deploy Notes
Import configuration as normal

### Testing on local
in the `docksal-local.env` file add/change the `PANTHEON_ENV` value to "pr-71" then run `fin init`. This will get the menu items from the test environment that cannot be added to DEV until the code is merged.
